packages/components/src/organisms/ImageGallery/ImageGallery.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes, FunctionComponent } from 'react'
+
+import { ImageGallerySelector } from '../..'
+
+type ImageComponentType = FunctionComponent<{
+  url: string
+  alternateName?: string
+  loading?: 'eager' | 'lazy'
+  onLoad?: (image: any) => void
+}>
+
+export type ImageElementData = {
+  /**
+   * Image URL.
+   */
+  url: string
+  /**
+   * Alternative text description of the image.
+   */
+  alternateName: string
```

Comment:
I wonder why not alt as img tag uses as default?

Reply:
I don't know why but seems like a pattern in project.